src/app/shared-services/db/sqlite.service.ts

```diff
@@ -126,9 +126,16 @@ export class SqliteService {
       message,
     } = await this.db.run({
       statement: 'INSERT OR REPLACE INTO points VALUES (?,?,?,?,?)',
-      values: [trajectoryId, time.toISOString(), ...p.latLng, p.accuracy],
+      // IDK why but with iOS simulator it was not possible to run this without .toFixed()
+      values: [
+        trajectoryId,
+        time.toISOString(),
+        p.latLng[1].toFixed(8),
+        p.latLng[0].toFixed(8),
```

Comment:
coords are in reverse order right? points table is sorted lat first, lon second

Reply:
Ah I missed that mapbox is using the reversed way. I will resolve that